<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgMatchIkFk</class>
 <widget class="QDialog" name="DlgMatchIkFk">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>124</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="cb_Side">
     <item>
      <property name="text">
       <string>Arm L</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Arm R</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Leg L</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Leg R</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cb_Match">
     <item>
      <property name="text">
       <string>Ik -&gt; Fk</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fk -&gt; Ik</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
